velodyne_pcl/package.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<package format="2">
```

Comment:
Please update to Format 3 and include the XML namespace declaration. e.g.

```
<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
```

Reply:
Thanks for adding the URL. However, you need to change the `<package>` tag to match (format="3").